api/src/main/resources/application.conf

```diff
@@ -5,6 +5,9 @@ geoprocessing {
 }
 
 akka.http {
+  server {
+    request-timeout = infinite
```

Comment:
We probably should not be doing this. The new end point does take on average 22s, the highest I've recorded is 35s, which is quite a bit more than the 20s default limit of Akka Http. I'm hoping we can do some tweaks to lower the time of the request, or at least add a more educated cap than `infinite`.